main/ertp/api/brand.md

```diff
@@ -21,8 +34,22 @@ const isIssuer = brand.isMyIssuer(issuer);
 ## brand.getAllegedName()
 - Returns: `{String}`
 
-Return the alleged name of the brand. Should not be trusted as accurate.
+Returns the alleged name of the `brand`. Should not be trusted as accurate.
+
+An alleged name is a human-readable string name of a kind of digital asset. 
+The alleged name should not be trusted as accurate, since it is not provided
+by a trusted source.
```

Comment:
I think this reason isn't quite right. It's that the allegedName is given by whoever called `makeIssuerKit` and there's no registry of names of mints, so there can be many mints of the same name, so the allegedName itself shouldn't be trusted to indicate identity.